test/functionalTest/cases/880_timeout_for_forward_and_notifications/notification_that_times_out.test

```diff
@@ -0,0 +1,153 @@
+# Copyright 2015 Telefonica Investigacion y Desarrollo, S.A.U
+#
+# This file is part of Orion Context Broker.
+#
+# Orion Context Broker is free software: you can redistribute it and/or
+# modify it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# Orion Context Broker is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero
+# General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with Orion Context Broker. If not, see http://www.gnu.org/licenses/.
+#
+# For those usages not covered by this license please contact with
+# iot_support at tid dot es
+
+# VALGRIND_READY - to mark the test ready for valgrindTestSuite.sh
+
+--NAME--
+Notification That Times Out
+
+--SHELL-INIT--
+dbInit CB
+brokerStart CB
+accumulatorStart
+
+--SHELL--
+
+#
+# 01. Register E1/T1/A1
+# 02. Availability-Subscribe to E1/A1 for accumulator, on a non-responding URL
+# 04. Sleep 6 secs (default timeout for notifications is 5 seconds)
+# 05. Grep in brokers logfile to see that the notification failed and that the broker thread continued
+# 06. Dump notifications in accumulator - see nothing
+#
```

Comment:
I would say that 90% of the notification use cases are for NGSI10. Thus, this test probably models a "closer reality" using NGSI10. However, probably doesn't worth the effort to create a new test if the functionality is already tested with this one (in this case, mark this comment as NTC).